package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-useasynccall",
+  "version": "1.0.0",
+  "description": "Our amazing useAsyncCall hook for React.js",
+  "main": "/lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
dat pryc

Reply:
prí to tam má být podle oficiálního videa aby bylo jasno že testy nemáš

Reply:
aha okej